packages/amplify_datastore/android/src/main/kotlin/com/amazonaws/amplify/amplify_datastore/types/model/FlutterModelSchema.kt

```diff
@@ -18,32 +18,33 @@ package com.amazonaws.amplify.amplify_datastore.types.model
 import com.amplifyframework.core.model.ModelSchema
 
 data class FlutterModelSchema(val map: Map<String, Any>) {
-    val name: String = map["name"] as String
-    private val pluralName: String = map["pluralName"] as String
-    private val authRules: List<FlutterAuthRule>? =
+    val name : String = map["name"] as String
+    private val pluralName : String? = map["pluralName"] as String?
+    private val authRules : List<FlutterAuthRule>? =
```

Comment:
Any reasons for adding a space before the colon in declaration? It is counter coding convention https://kotlinlang.org/docs/reference/coding-conventions.html#colon

Reply:
No good reason, thank you for pointing that out.  I've updated the method signatures and typing to follow that format.